pkg/testsuite/mock/node.go

```diff
@@ -131,35 +137,35 @@ func (n *Node) hookEvents() {
 	events.MainEngineSwitched.Hook(func(e *engine.Engine) { n.mainEngineSwitchedCount.Add(1) })
 }
 
-func (n *Node) HookLogging() {
+func (n *Node) hookLogging(failOnBlockFiltered bool) {
 	events := n.Protocol.Events
 
-	n.attachEngineLogs(n.Protocol.MainEngineInstance())
+	n.attachEngineLogs(failOnBlockFiltered, n.Protocol.MainEngineInstance())
 
 	events.Network.BlockReceived.Hook(func(block *model.Block, source identity.ID) {
-		fmt.Printf("%s > Network.BlockReceived: from %s %s - %d\n", n.Name, source, block.ID(), block.ID().Index())
+		fmt.Printf("%s > network.BlockReceived: from %s %s - %d\n", n.Name, source, block.ID(), block.ID().Index())
```

Comment:
why lowercased network? IDE rename strikes again?

Reply:
Seems like it